app/views/devise/registrations/_address_form.html.haml

```diff
@@ -0,0 +1,17 @@
+= form.fields_for :adress do |address|
+  .form-group.row
+    = label_helper(address, :city, 'city-input')
```

Comment:
Let's discuss the approach to invoking form helpers. I would like to share a few best practices ;-)